.travis.yml

```diff
@@ -2,18 +2,19 @@ language: c
 sudo: required
 install:
   - wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-opam.sh
-  - wget https://raw.githubusercontent.com/simonjbeaumont/ocaml-travisci-skeleton/docgen/.travis-docgen.sh
+  - wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-docgen.sh
 script:
   - bash -ex .travis-opam.sh
-  - ( ( test $TRAVIS_PULL_REQUEST = "false" && test $TRAVIS_BRANCH = "master" && bash -ex .travis-docgen.sh ) || true )
+after_success:
+  - test "$GEN_DOC" = 1 && test $TRAVIS_PULL_REQUEST = "false" && test $TRAVIS_BRANCH = "master" && bash -ex .travis-docgen.sh
```

Comment:
I left it like this so we get an error when they are not defined, because the logic of this line depends on their existence, so I thought it's useful if we get an error if they don't exist.